.bash_nav

```diff
@@ -7,3 +7,31 @@ alias desk='cd /mnt/c/Users/gsera/Desktop'
 # source bash
 
 alias srcbash='source ~/.bashrc'
+
+# ---This alias effectively creates a bookmarking app---
+# it adds global symlink shortcuts for use with the 'cds' command
+# ex: 'cds studycloud' will run 'cd -P ~/.short/studycloud'
+# to add a shortcut: go to the target directory and run 'cds + <shortcut_name>'
+# to remove a shortcut: run 'cds - <shortcut_name>'
+cds() {
+    if [ ! -d ~/.short ]; then
+        mkdir ~/.short
+    fi
+    if [ $# -eq 0 ]; then
+        ls ~/.short/
+    else
+        if [ $1 = "+" ]; then
+            ln -s "$PWD" "$HOME/.short/$2"
+        elif [ $1 = "-" ] && [ $# -eq 2 ] && [ -L ~/.short/$2 ]; then
+            rm ~/.short/$2
+        else
+            if [ -d ./$1 ] || [ $1 = "-" ]; then
```

Comment:
If the shortcut doesn't exist, it's likely that the user forgot to type cd instead of cds. Let's check whether a directory like this exists and cd into it to save them some time.
It might also be that the user typed `cds -`, which should return to the last directory, just like `cd -`.